src/js/common/components/Modal.jsx

```diff
@@ -31,7 +34,10 @@ class Modal extends React.Component {
 
   componentDidUpdate(prevProps) {
     if (!prevProps.visible && this.props.visible) {
-      document.querySelector('.va-modal').focus();
+      const focusableElement = document.querySelector(this.props.focusSelector);
```

Comment:
Similar question as above, should this be limited to the modal itself rather than the whole document?